src/ast/value.rs

```diff
@@ -32,6 +32,9 @@ pub enum Value {
     #[cfg(not(feature = "bigdecimal"))]
     Number(String, bool),
     #[cfg(feature = "bigdecimal")]
+    // HINT: use `test_utils::number` to make an instance of
```

Comment:
I figured I would leave a breadcrumb to help future users find `test_utils:::number`